api/src/main/resources/workbench.yaml

```diff
@@ -184,6 +276,43 @@ definitions:
     description: levels of access to data in the curated data repository
     enum: &DATA_ACCESS_LEVEL [registered, protected]
 
+  WorkspaceListResponse:
+    type: "object"
+    required:
+     - "items"
+    properties:
+      items:
+        type: "array"
+        items:
+           $ref: "#/definitions/Workspace"
+
+  Workspace:
+    type: "object"
+    required:
+     - "name"
+     - "cohorts"
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      namespace:
+        type: string
+      description:
+        type: string
+      creator:
+        type: string
+      creationTime:
+        type: string
+        format: date-time
+      lastModifiedTime:
+        type: string
+        format: date-time
```

Comment:
I'm going to punt on that for now, but I think following the wireframe it would need to be free text because there's a free-text option. (With us providing suggestions)